roll/http.py

```diff
@@ -334,6 +334,10 @@ async def write(self, *args):
         else:
             if not self.parser.should_keep_alive():
                 self.transport.close()
+        # Transport is not reading, we assume an error has raised before the request
+        # has been fully read (missing a call to transport.resume_reading).
+        if not self.transport.is_reading():
```

Comment:
Tempted to rely on `transport._paused` instead.

It's a private API, but I think it's more what we want to track.

See https://github.com/python/cpython/blob/e0bec69854ff17b1a6db9bc01db8c002761bb7a8/Lib/asyncio/selector_events.py#L792-L793
```
    def is_reading(self):
        return not self._paused and not self._closing
```

Reply:
I agree, let's use it with a comment 😅